packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx

```diff
@@ -1,35 +1,142 @@
 import * as React from 'react';
-import {styled, themeGet} from '@twilio-paste/styling-library';
-import {Anchor} from '@twilio-paste/anchor';
-import {Box} from '@twilio-paste/box';
-import {Text} from '@twilio-paste/text';
-import {Heading} from '@twilio-paste/heading';
-import {Paragraph} from '@twilio-paste/paragraph';
-import {TWILIO_BLUE} from '../../constants';
-import {TwilioIcon} from '../icons/TwilioIcon';
-
-const StyledFooter = styled.footer`
-  margin-top: 96px;
-  padding: ${themeGet('space.space150')};
-  background-color: ${themeGet('colors.colorGray10')};
-`;
-
-const StyledFooterLogo = styled(TwilioIcon)`
-  margin: 0 auto ${themeGet('space.space40')};
-`;
+import {useTheme} from '@twilio-paste/core/theme';
+import {Anchor} from '@twilio-paste/core/anchor';
+import {Box} from '@twilio-paste/core/box';
+import {Stack} from '@twilio-paste/core/stack';
+import {Text} from '@twilio-paste/core/text';
+import HeartDoodleIcon from '../icons/HeartDoodleIcon';
+import TwilioLogo from '../icons/TwilioLogo';
+import FooterBuildingBlocks from '../../assets/illustrations/FooterBuildingBlocks.svg';
 
 const SiteFooter: React.FC<{}> = () => {
+  const theme = useTheme();
   return (
-    <StyledFooter>
+    <Box
```

Comment:
nit: I don't _love_ these large blocks of nested boxes in general. A little part of me still likes to clean stuff up like this, just like I would in the old BEM days, creating component names for different blocks, and component files.

Reply:
I broke the footer out into separate components files so they're much smaller chunks of code.